hpy/devel/include/universal/hpy.h

```diff
@@ -18,7 +18,8 @@ typedef intptr_t HPy_ssize_t;
 typedef intptr_t HPy_hash_t;
 struct _HPy_s { HPy_ssize_t _i; };
 typedef struct _HPy_s HPy;
-typedef struct { HPy _lst; } HPyListBuilder;
+typedef struct { HPy_ssize_t _lst; } HPyListBuilder;
```

Comment:
maybe add a comment explaining that `_lst` is just an opaque pointer-like field which is implementation-specific. Else people might think that it contains the length or something like that.